src/static/css/main.less

```diff
@@ -4,16 +4,16 @@
    ========================================================================== */
 
 // import Normalize before all other LESS/CSS
-@import "../../vendor/normalize-css/normalize.css";
+@import (less) "../../vendor/normalize-css/normalize.css";
```

Comment:
We did this intentionally on cfgov-refresh for reasons that elude me at the moment. Let me see if I can dig up that discussion.